app/models/cdr/cdr.rb

```diff
@@ -162,6 +162,7 @@ class Cdr::Cdr < Cdr::Base
   ].freeze
 
   include Partitionable
+  include RoutingTagIdsScopeable
```

Comment:
you can copy `scope :tagged, lambda { |value| ... }` here instead of including module because second scope is not needed